os/drivers/lwnl/lwnl.c

```diff
@@ -42,15 +42,17 @@
 
 #define LWNLDEV_LOCK(upper)						\
 	do {										\
-		ret = sem_wait(&upper->exclsem);		\
-		if (ret < 0) {							\
+		int lock_ret = sem_wait(&upper->exclsem);		\
+		lldbg("[lwnl] lock (%p)\n", &upper->exclsem);	\
```

Comment:
Sometimes this log will be printed after unlocking the semaphore, because if `upper->exclsem` is available, `sem_wait()` 
 will block the context directly. 
Is it okay?